libnd4j/include/cblas.h

```diff
@@ -21,7 +21,9 @@
 #ifndef NATIVEOPERATIONS_CBLAS_H
 #define NATIVEOPERATIONS_CBLAS_H
 
+#ifndef __STANDALONE_BUILD__
 #include "config.h"
```

Comment:
You're going to need BLAS, MKL-DNN, etc even in "standalone" mode.

Reply:
Once we need it - we'll add it immediately.